.github/workflows/release.yml

```
@@ -0,0 +1,122 @@
+on:
+  push:
```

Adding `workflow_dispatch` will allow you to trigger it manually, in case the build fails.

```suggestion
  workflow_dispatch:
  push:
```